services/core-web/src/components/Forms/ExplosivesPermit/MagazineFormNew.tsx

```diff
@@ -224,29 +235,31 @@ class MagazineForm extends Component<FormProps> {
           <Col md={24}>
             {fields.map((field, index) => (
               <Collapse
-                key={field.id}
+                key={field}
```

Comment:
totally unrelated to what I was doing: I kept getting duplicate key warnings in my console, and it turns out that **field does not have an id property**. So we should fix that. Field is a string.